lib/bootloader_setup.pm

```diff
@@ -554,8 +554,9 @@ sub specific_bootmenu_params {
             # due to BSC#932692 (SLE-12). 'SetHostname=0' has to be set because autoyast
             # profile has DHCLIENT_SET_HOSTNAME="yes" in /etc/sysconfig/network/dhcp,
             # 'ifcfg=*=dhcp' sets this variable in ifcfg-eth0 as well and we can't
-            # have them both as it's not deterministic.
-            $netsetup = get_var("NETWORK_INIT_PARAM", "ifcfg=*=dhcp SetHostname=0");
+            # have them both as it's not deterministic. Don't set on IPMI with net interface defined in SUT_NETDEVICE.
+            my $ifcfg = check_var('BACKEND', 'ipmi') ? '' : 'ifcfg=*=dhcp SetHostname=0';
```

Comment:
so you dropping just default value in case of IPMI , but if someone will defaine NETWORK_INIT_PARAM for IPMI it will ignore your logic. 
So for me it would be more clear to guard $args variable instead of $ifcfg with same check, so we **really** making sure that we not using this with IPMI: 
```suggestion
            if(!check_var('BACKEND','ipmi')) {
                 $netsetup = get_var("NETWORK_INIT_PARAM", "ifcfg=*=dhcp SetHostname=0") ;
                 $args .= " $netsetup ";
             }
```

Reply:
if somobody will use NETWORK_INIT_PARAM with IPMI then it will be used instead of default value http://10.100.12.155/tests/9457#step/boot_from_pxe/6